### ISSUE TITLE
why only 49-batches for weights?

### ISSUE THREAD

**Issue by 777Adonis** [elvisyjlin/AttGAN-PyTorch#17]
Thank you for your contribution, but your pre-training weights file has only 49 batches. I want to know if you have used the learning rate attenuation strategy?

**Comment by elvisyjlin** [comment]
You're welcome. Sorry for my late reply. I was busy in the past 3 months.

You probably meant `weights.49.pth`, which is the model weights at the _49th_ epoch. We apply the same learning rate attenuation strategy as the code by the original author. The model is updated with base learning rate in the first 100 epochs whilst the learning rate is halved in the last 100 epochs.

However, the whole training is too long. It took about 2 weeks to train under one setting on my machine. For the sack of time, we simply train each model for 49 epochs. Please do the training yourself to get the 200-epoched model. Yet, believe me, the performance is actually good enough with 49 epochs.
